src/app/scheduler/components/SchedulerSections.tsx

```diff
@@ -0,0 +1,59 @@
+import { Radio, RadioGroup, Thead, Table, Tbody, Tr, Th, Td } from '@chakra-ui/react';
+import React from 'react';
+
+import { ClassScheduleListing, MeetingTimes } from '../../../shared/fetchers';
+
+export function Sectionss({ sections }: { sections: ClassScheduleListing[] }): JSX.Element {
+  const [section, setSection] = React.useState('');
+  return (
+    <RadioGroup onChange={setSection} value={section}>
+      <Table size="sm">
+        <Thead>
+          <Tr>
+            <Th>Option</Th>
+            <Th>Section</Th>
+            <Th>Days</Th>
+            <Th>Times</Th>
+            <Th>Location</Th>
+          </Tr>
+        </Thead>
+        <Tbody>
+          {sections.map(({ sectionCode, meetingTimes }) => (
+            <Option sectionCode={sectionCode} meetingTimes={meetingTimes} />
+          ))}
```

Comment:
So this is where the complexity lies... 
If you think about it, you should only be able to select 1 section / section type. This means we obviously can't select two lecture sessions (ie you don't register for `A01` and `A02`). But if there are tutorials and labs, we want to select at least one of them. I don't think this is going to enforce that rule.